start-mycroft.sh

```diff
@@ -77,51 +78,54 @@ function name-to-script-path() {
     esac
 }
 
-function source-venv() {
-    # Enter Python virtual environment, unless under Docker
```

Comment:
what is the requirement/reasoning for not using python venv within docker? (is it pre-loaded?)

Reply:
If I recall correctly they wanted to skip this step in the docker since it isn't strictly needed (the docker environment is enough of a sandbox). However I believe the current docker setup (in [docker-mycroft](https://github.com/mycroftai/docker-mycroft) is currently using the venv